stl/inc/chrono

```diff
@@ -5486,34 +5499,61 @@ struct formatter<_CHRONO day, _CharT> {
         }
 
         for (const auto& _Spec : _List) {
-            if (_Spec._Type != '\0' && _Spec._Type != 'd' && _Spec._Type != 'e') {
-                _THROW(format_error("Invalid type for type day."));
-            }
-            if (_Spec._Modifier != '\0' && _Spec._Modifier != 'O') {
-                _THROW(format_error("Invalid modifier for type day."));
+            if (_Spec._Type != '\0' && _RANGES find(_Valid_types, _Spec._Type) == _Valid_types.end()) {
+                _THROW(format_error("Invalid type."));
             }
+            _Check_modifier(_Spec._Type, _Spec._Modifier);
         }
 
         return _Res_iter;
     }
 
-    template <class _FormatContext>
-    typename _FormatContext::iterator format(const _CHRONO day& _Val, _FormatContext& _FormatCtx) {
-        using _Ctx_iter = typename _FormatContext::iterator;
+    void _Check_modifier(const _CharT _Type, const _CharT _Modifier) {
+        if (_Modifier == _CharT{'\0'}) {
+            return;
+        }
+
+        enum _Allowed_bit { _E = 1, _O = 2, _EO = _E | _O };

```

Comment:
Nitpick: These are macro-like (underscore and all-caps), which we conventionally avoid.